### PR TITLE
Add getters to FileProvider

### PR DESCRIPTION
**Pull request by tifabien**
I am targeting this branch, because it adds helpful getters to FileProvider class.

## Changelog

```markdown
### Added
- Added getters for `allowedExtensions` and  `allowedMimeTypes` properties
```

Useful when we inject the provider in twig.

### CODE REVIEW

**Comment by OskarStark** [comment]
Could you please add a test case and do we need to change/extend an Interface?

**Comment by kunicmarko20** [comment]
There is no interface and these properties are only in FileProvider, about the Tests, do we really need to test getters? :smile:

File: src/Provider/FileProvider.php [hunk 1]
```diff
@@ -88,6 +88,22 @@ public function getReferenceFile(MediaInterface $media)
         return $this->getFilesystem()->get($this->getReferenceImage($media), true);
     }
 
+    /**
+     * @return array
```
Comment (greg0ire) [inline src/Provider/FileProvider.php]: please use the `itemType[]` notation

**Comment by OskarStark** [comment]
> There is no interface and these properties are only in FileProvider, about the Tests, do we really need to test getters? 😄

I was on mobile yesterday and I am fine without the tests, but I really do getter tests because of misconfigured setters mostly ;-)

After @tifabien applied @greg0ire 's comment, we are ready to merge 👍

**Pull request merged**